package.json

```diff
@@ -230,7 +228,6 @@
 		"redux": "^5.0.1",
 		"request": "^2.88.2",
 		"swiper": "^10.1.0",
-		"workspace": "^0.0.1-preview.1",
```

Comment:
Lol, seems like it was an unintended install (#83862), and a [very common](https://www.npmjs.com/package/workspace) one.

Reply:
Good find!